truss/templates/base.Dockerfile.jinja

```diff
@@ -17,8 +17,11 @@ RUN $PYTHON_EXECUTABLE -c "import sys; \
 {% endblock %}
 
 {% block install_uv %}
-{# Install `uv` if not already present in the image. #}
-RUN if ! command -v uv >/dev/null 2>&1; then (curl -LsSf https://astral.sh/uv/{{ UV_VERSION }}/install.sh | sh) >/dev/null 2>&1; fi
+{# Install `uv` and `curl` if not already present in the image. #}
+RUN if ! command -v uv >/dev/null 2>&1; then \
```

Comment:
Initially I had wrapped the uv installation in additional pipe to /dev/null so we don't see some of that output, but it did silence the underlying error here. I don't think users will care about the ~5 lines of output, and we can always silence once this is more battle tested